scripts/util.py

```diff
@@ -0,0 +1,36 @@
+import os
+from pathlib import Path
+
+from constants import IS_ACTIONS
+
+def get_major_version(version: str) -> str:
```

Comment:
I would have much preferred a class-based approach to the 'Format Version. You can see my implementation in Reticulator for example. 

Once you have it as a class, you can define methods such as the ones found in this utils file. It's the same philosophy as Pathlib, which I see you're using.

Reply:
Great idea! Reduced unnecessary runs of functions and cleaned up a lot as well.